### PR TITLE
Node scripts in content directory

### PR DESCRIPTION
**Pull request by mamaso**
I'll check the zip artifacts on appveyor before merging this in

### CODE REVIEW

**Comment by mathewc** [comment]
Please also take the private site extension package from artifacts, and verify it runs by deploying it (instructions [here](https://github.com/Azure/azure-webjobs-sdk-script/wiki#option-1-http-functionwebhook-support)).

File: WebJobs.Script.proj [hunk 1]
```diff
@@ -110,9 +110,11 @@
     <ItemGroup>
       <ScriptHostBinaries Include="$(ScriptHostOutput)\*.dll;$(ScriptHostOutput)\*.exe*;$(ScriptHostOutput)\*.optdata;$(ScriptHostOutput)\*.sigdata"/>
       <EdgeJsFiles Include="$(ScriptHostOutput)\edge\**\*.*"/>
+      <StaticContent Includ="$(ScriptHostOutput)\Content**\*.*">
```
Comment (mathewc) [inline WebJobs.Script.proj]: "Includ" - this can't work :)

Comment (fabiocav) [inline WebJobs.Script.proj]: And just to make sure, "inclo" won't work either.

**Comment by mamaso** [comment]
@mathewc Content folder is in all zips and the private site extension runs, should be good to go!

**Pull request merged**